examples/00-fluent/external_compressible_flow.py

```diff
@@ -301,9 +302,9 @@
 
 pressure_farfield.flow_direction[2] = 0.053382
 
-pressure_farfield.turb_intensity = 0.05
+pressure_farfield.turbulent_intensity = 0.05
 
-pressure_farfield.turb_viscosity_ratio = 10
+pressure_farfield.turbulent_viscosity_ratio_real = 10
```

Comment:
@mkundu1 @seanpearsonuk Any idea why this has the strange `_real` suffix on it now.  It was fine before.

Reply:
See #2251